src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,9 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User byEmail = userService.findByEmail(email);
```

Comment:
a bad name for a user variable
let's just make it a `user`
```suggestion
        User user = userService.findByEmail(email);
```